packages/legend-graph/src/graph-manager/__test-utils__/EngineTestSupport.ts

```diff
@@ -34,7 +34,7 @@ import {
   type V1_ArtifactGenerationExtensionOutput,
 } from '../protocol/pure/v1/engine/generation/V1_ArtifactGenerationExtensionApi.js';
 
-export const ENGINE_TEST_SUPPORT_API_URL = 'http://localhost:6300/api';
+export const ENGINE_TEST_SUPPORT_API_URL = 'http://localhost:9006/api';
```

Comment:
this is not right, the backend we use for testing is engine server at port `6300`, this test utility is shared with other parts of the codebase, not just REPL, if you need something, see if you can use engine's APIs instead (pretty sure to compose ValueSpec, you can)